packages/api/src/api/project/post.ts

```diff
@@ -25,6 +26,10 @@ export const post = async (req: Request, res: Response) => {
 
       project = new Project(data);
       project.contributors.add(lockedUser);
+      const hostRes = await axios.get(data.repoLink);
```

Comment:
This is unrelated to the line above so let's break it up a bit
```suggestion

      const hostRes = await axios.get(data.repoLink);
```

Also what is `hostRes`? Are you fetching a host? – please rename to something that's a bit more descriptive like `repoFetchRes`

Finally you need a `try ... catch` around this fetch

Reply:
A try inside of a try?

Reply:
Yep good call on the try; the name of the variable still needs to change though